cityofaustin/atd-moped
# Remove phase name from moped_proj_phases

## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/9350

This is the second part of normalizing the moped_phases and moped_proj_phases tables. In this PR I have updated the project_list_view to not rely on the phase_name from moped_proj_phases and updated the Signal Projects table query as well. 

I also ripped the band aid and dropped the phase_name column. Ive gone through multiple times looking for references to the phase_name column on the moped_proj_phases table in the editor and think I removed them all, but would really appreciate some kicking of the tires. 

## Testing
**URL to test:** 
https://9350-normal-phase--atd-moped-main.netlify.app/moped/projects

**Steps to test:**

Add a construction phase (with start date) or completion phase (with end date) to a project. 
Check that the dates appear in the projects list. 

Add a project with a signal, add a phase and mark as current. Go to the signals view and see the name of the current phase. 
https://9350-normal-phase--atd-moped-main.netlify.app/moped/views/signal-projects

_There should not be any change in functionality from existing behavior in moped._ 


---
#### Ship list
